projecttacoma/deqm-test-kit
# Upgrade to Inferno Core 0.0.7

# Summary
Updated our Inferno Core version to 0.0.7

## New behavior
Hopefully none except some bug fixes in the UI, and immediately starts a test session with DEQM Measure Operation Test Suite

## Code changes
Updated Inferno Core version. Added additional dependencies to `spec_heper.rb` file to repair unit tests broken by upgrade

# Testing guidance

- run `bundle install`
- run the rspec tests
- spin up the test server using `docker-compose pull` and then `docker-compose up --build`
- Run all tests in the Suite! Keep in mind that we are expecting the first dataRequirements test to fail when using deqm-test-server and we expect the second data requirements test and the second and third submit data tests to fail when using cqf-ruler.
